pimatic-led-light.coffee

```diff
@@ -17,9 +17,9 @@ module.exports = (env) ->
     init: (app, @framework, @config) =>
       deviceConfigDef = require("./led-light-schema")
 
-      @framework.deviceManager.registerDeviceClass "LedLight",
+      @framework.deviceManager.registerDeviceClass "IWYLed",
```

Comment:
Yes. But please Note. I just made this for The general idea. So i probably broke almost erything with this pull ;)